### ISSUE TITLE
US oilers bugged (either fuel draw or tank capacity)

### ISSUE THREAD

**Issue by FIM43-Redeye** [PygmalionOfCyprus/cmo-db-requests#1334]
### DB Selector

CWDB

### Affected DBID(s)

Most US oilers

### Description of Problem

For context, there seems to be some kind of pervasive DB bug- I don't know much about how fast ships burn fuel, so I'm laying out the problem at hand.

As an example, B 16 Punta Delgada, an Argentinian navy ship, burns 1kg of fuel per minute at cruise and contains 66.7 tons of diesel fuel. This sounds absurdly low for a ship, and I am assuming the US oilers have more reasonable fuel draw - such as AO 105 Mispillion, which carries 18,560 tons of fuel and burns 306kg per minute at cruise. However, a significant number of oilers have messed-up fuel quantities: AO 129 Maumee has only 57 tons of oil fuel, 8.5 tons of aviation fuel, and 1 ton of diesel and gas fuel, but burns 325kg per minute at cruise.

### Recommended Fix

Either give these ships way more fuel or pull the fuel consumption way down. If CV 42 eats one kilo per minute and has 36 tons of oil fuel, it makes no sense for an oiler to have barely double that and burn 300 times more.

**Comment by PygmalionOfCyprus** [comment]
Ugh. CWDB has been plagued by propulsion issues since before my time...just when I think I've got it dealt with, something new pops up. Or a fix to something breaks something else.

Out of curiosity, does this issue persist in older CWDBs? (Real old ones, like 479, etc.) If yes then I'm going to have to go through and assign primary fuel values. If no, I can steal the old "working" numbers and apply them to the current version.

**Comment by FIM43-Redeye** [comment]
Issue does not appear to persist in older CWDBs, although the previously brought up (I think) issue of carriers barely sucking any fuel is still present. My oldest DB is v456, and the newest one without the issue is v478 (that I have). My guess is CWDB needs a common-sense overview for every ship, or maybe a utility to calculate time on station so one can just glance at it and determine which are messed up? All conjecture here, I'm no database builder, but a quick script could probably go a long way in that regard.

**Comment by PygmalionOfCyprus** [comment]
Yes, it does, and that's probably how I'm going to tackle it once we get the bandwidth. I'll try not to let this languish too long.

**Comment by PygmalionOfCyprus** [comment]
While investigating a related issue I discovered what I believe to be the cause of this, or at least of a large number of propulsion errors in CWDB. I'll spare the gory details, but it seems that for some time (like, years) a large number of the CWDB propulsion entries' fuel consumption was based on incorrect numbers, resulting in absurd burn rates; these all needed to be recalculated (which we have an extant utility for) but somehow it was never done. I've just done it.

This won't fix _everything_. A lot of propulsion entries in CWDB seem to be double-dipped, which means the same engine is being used on different ships with wildly different specs. That has to be fixed too, though it'll require more work.

But this is something. The battle continues.
